server/integration_tests/test_data/e2e_date_range/whichcountriesinafricahavehadthegreatestincreaseinelectricityaccessoverthelast10years/chart_config.json

```diff
@@ -10,20 +10,23 @@
                 "tiles": [
                   {
                     "comparisonPlaces": [
-                      "country/AGO",
-                      "country/CIV",
-                      "country/CMR",
-                      "country/BEN",
-                      "country/COD",
-                      "country/BDI",
-                      "country/DZA"
+                      "country/KEN",
+                      "country/RWA",
+                      "country/ETH",
+                      "country/UGA",
+                      "country/TZA",
+                      "country/MLI",
+                      "country/ZMB",
+                      "country/SDN",
+                      "country/ZWE",
+                      "country/AGO"
```

Comment:
Is this change from the fact that we use facet-ids from within the time-delta handlers?